app/models/user.rb

```diff
@@ -221,7 +221,11 @@ def can_manage_user?(user)
   end
 
   def enabled_for_organization?(org)
-    user_organizations.find_by(organization: org).try(:enabled?)
+    user_organizations.where(enabled: true, organization_id: org.id).exists?
+  end
+
+  def enabled_for_market?(market)
```

Comment:
not sure of LO policy on this.. I thought this was necessary when trying to fix a spec, but it turned out that my spec was wrong (ie: the test user's association to their first organization in the market was being set to false,  but turns out they had *two* organizations and the other meant that sending the sheet to them was expected...)

Reply:
todo - probably nuke.